src/authInfo.ts

```diff
@@ -832,6 +839,12 @@ export class AuthInfo extends AsyncCreatable<AuthInfo.Options> {
 
       if (this.isTokenOptions(options)) {
         authConfig = options;
+        const userInfo = await this.retrieveUsername(
```

Comment:
For access token access, retrieve username with access token and capture username and orgid.